ydb/docs/en/core/integrations/visualization/superset.md

```diff
@@ -1,13 +1,95 @@
 # Apache Superset
 
-Apache Superset is a modern data exploration and data visualization platform.
+[Apache Superset](https://superset.apache.org/) is a modern data exploration and data visualization platform. This article will explain how to build visualizations on top of the data stored in YDB.
 
-[PostgreSQL compatibility mode in {{ ydb-short-name }}](../../postgresql/intro.md) enables the use of [Apache Superset](https://superset.apache.org/) to query and visualize data from {{ ydb-short-name }}. In this case Apache Superset works with {{ ydb-short-name }} just like with PostgreSQL.
+## Installation of dependencies {#prerequisites}
 
-{% include [../../postgresql/_includes/alert_preview.md](../../postgresql/_includes/alert_preview.md) %}
+To work with {{ ydb-short-name }} via Superset, you need to install the [ydb-sqlalchemy](https://pypi.org/project/ydb-sqlalchemy) driver.
+
+The installation method depends on how Superset is set up. For detailed guidance, refer to the [official documentation](https://superset.apache.org/docs/configuration/databases/#installing-drivers-in-docker-images).
 
 ## Adding a database connection to {{ ydb-short-name }} {#add-database-connection}
 
+There are two ways to connect to {{ ydb-short-name }}:
+
+1. Native connection using sqlalchemy driver (starting from version 5.0.0)
```

Comment:
I wanted to suggest adding a link to documentation about our integration with sqlalchemy, and it's not in https://ydb.tech/docs/en/integrations/orm/?version=main for some reason.